.github/workflows/check-full.yaml

```diff
@@ -25,9 +25,9 @@ jobs:
           # use 4.1 to check with rtools40's older compiler
           - { os: windows-latest, r: "4.1" }
 
-          - { os: ubuntu-latest, r: "devel", http-user-agent: "release" }
-          - { os: ubuntu-latest, r: "devel", locale: "en_US" }
+          - { os: ubuntu-latest, r: "devel", locale: "en_US", http-user-agent: "release" }
           #- { os: ubuntu-latest, r: "release", locale: "zh_CN" }
+          - { os: ubuntu-latest, r: "release" }
```

Comment:
This was unintentionally removed in the PR that updated to support difference locales. So adding it back.